image/share/Taskfile

```diff
@@ -0,0 +1,3 @@
+default: k=64
+adaptive: k=$(estimate_kmer.sh ${READS})
+careful: k=96
```

Comment:
In my experience, larger values of *k* are more likely to create misassemblies than smaller values of *k*, so "careful" is a bit of a misnomer. I'd call this option "k96".